src/components/wizard-upload-configfile/wizard-upload-configfile.js

```diff
@@ -0,0 +1,44 @@
+import {copy, fromJson, module} from 'angular';
+
+export default module('app.components.wizard-upload-configfile', [])
+  .component('wizardUploadConfigfile', {
```

Comment:
This component looks fairly generic now and maybe we also want to use it outside of the wizard or for other than config files. What do you think about renaming it to `load-file-form` (it basically wraps `load-file-button` for use in a form)?